src/__tests__/Layout-test.js

```diff
@@ -1586,5 +1587,78 @@ describe('Layout', function() {
     );
   });
 
+});
+
+describe('Layout alignContent', function() {
```

Comment:
Can you figure out why those tests are not being transpiled to C and Java? Is it because you are creating a new describe?